reference/api/dump.md

```diff
@@ -28,35 +28,12 @@ The task queue will not process any further tasks during dump creation, but you
 
 ```json
 {
-  "uid": "20200929-114144097",
-  "status": "in_progress",
-  "startedAt": "2020-09-29T11:41:44.392327Z"
+  "taskUid": 0,
```

Comment:
```suggestion
  "taskUid": 1,
```

I think we agreed to set any `taskUid` that wasn't creating an index to 1

Reply:
FWIW I think that specific discussion was about the quick start, where we actually have some knowledge of how many tasks the user has created up to that point; here, we don't. Still, this rule doesn't seem harmful so I personally am fine with it.